cli/internal/graph/graph.go

```diff
@@ -61,8 +64,13 @@ func (g *CompleteGraph) GetPackageTaskVisitor(ctx gocontext.Context, visitor fun
 			TaskDefinition:  taskDefinition,
 			Outputs:         taskDefinition.Outputs.Inclusions,
 			ExcludedOutputs: taskDefinition.Outputs.Exclusions,
+			ExpandedInputs:  make(map[turbopath.AnchoredUnixPath]string),
 		}
 
+		pfs := taskhash.SpecFromPackageTask(packageTask)
+		expandedInputs := g.TaskHashTracker.PackageInputsExpandedHashes[pfs.ToKey()]
+		packageTask.ExpandedInputs = expandedInputs
```

Comment:
I think we're exposing a lot of `taskhash` internals just for these three lines. I think we can keep all of the internals private if we add a method to the tracker that does this:
```
func (th *Tracker) GetExpandedInputs(packageTask *nodes.PackageTask) map[turbopath.AnchoredUnixPath]string {
	pfs := specFromPackageTask(packageTask)
	expandedInputs := th.packageInputsExpandedHashes[pfs.ToKey()]
	inputsCopy := make(map[turbopath.AnchoredUnixPath]string, len(expandedInputs))
	for path, hash := range expandedInputs {
		inputsCopy[path] = hash
	}

	return inputsCopy
}
```

If something down the line requires exposing all of these internals then these are some follow ups:
 - I really don't know what the "Go" way is, but should we make `ExpandedInputs` a pointer so we're more explicit that the map is shared between multiple structs? Either that or we should copy the map here: https://github.com/uber-go/guide/blob/master/style.md#copy-slices-and-maps-at-boundaries
 - From reading through the code I think we're okay with `packageTask.ExpandedInputs` being `nil` so we can have the assignment on the same line as the lookup instead of using a local variable.

Reply:
Great idea. The public method makes more sense. I can always expose pieces for future work, but I don't think we should do it here either